packages/build-tools/src/steps/functionGroups/build.ts

```diff
@@ -0,0 +1,149 @@
+import { BuildFunctionGroup, BuildStep, BuildStepGlobalContext } from '@expo/steps';
+import { Platform } from '@expo/eas-build-job';
+
+import { createCheckoutBuildFunction } from '../functions/checkout';
+import { createInstallNodeModulesBuildFunction } from '../functions/installNodeModules';
+import { createPrebuildBuildFunction } from '../functions/prebuild';
+import { createInstallPodsBuildFunction } from '../functions/installPods';
+import { configureEASUpdateIfInstalledFunction } from '../functions/configureEASUpdateIfInstalled';
+import { generateGymfileFromTemplateFunction } from '../functions/generateGymfileFromTemplate';
+import { runFastlaneFunction } from '../functions/runFastlane';
+import { createFindAndUploadBuildArtifactsBuildFunction } from '../functions/findAndUploadBuildArtifacts';
+import { CustomBuildContext } from '../../customBuildContext';
+import { createGetCredentialsForBuildTriggeredByGithubIntegration } from '../functions/getCredentialsForBuildTriggeredByGitHubIntegration';
+import { resolveAppleTeamIdFromCredentialsFunction } from '../functions/resolveAppleTeamIdFromCredentials';
+import { configureIosCredentialsFunction } from '../functions/configureIosCredentials';
+import { runGradleFunction } from '../functions/runGradle';
+import { configureIosVersionFunction } from '../functions/configureIosVersion';
+import { injectAndroidCredentialsFunction } from '../functions/injectAndroidCredentials';
+import { configureAndroidVersionFunction } from '../functions/configureAndroidVersion';
+
+interface HelperFunctionsInput {
+  globalCtx: BuildStepGlobalContext;
+  buildToolsContext: CustomBuildContext;
+}
+
+export function createEasBuildBuildFunctionGroup(
```

Comment:
And _now_ we can delete the non-custom builds flow haha!

Reply:
hahah xD